docs_styleguide.md

```diff
@@ -0,0 +1,240 @@
+# AWS CLI code example style guide
+
+```{danger}
+The AWS CLI repository is a **public** GitHub repository and submitting examples before a feature launch produces issues. You must be able to run the command from the public installation source.
+```
+
+This page outlines the code example guidelines for AWS Command Line Interface (AWS CLI) examples as part of the [contribution process](contribution_process.md). 
+
+-----
+
+## Before you start
+
+* Create all examples using a plain text editor or code editor like Notepad, Notepad++, or VS Code. **Do not** use word processors like Word or Outlook. We suggest you use an editor where you can easily see whitespace.
+* Always [sanitize sensitive information](docs_styleguide#sanitizing-sensitive-information) in your examples.
+
+## Example file (template)
+
+Download the `.rst` template: [command-name.zip](_static/files/command-name.zip)
```

Comment:
I'm wondering if we should just make the static file an `.rst` file instead of a zip? It felt unnecessary that it was a zip. Also it is easier to review future changes to the file as it will not be a binary when rendered in diffs.